include/internal/quic_wire.h

```diff
@@ -116,7 +116,7 @@ typedef struct ossl_quic_frame_ack_st {
      * acknowledge at least one packet number.
      */
     OSSL_QUIC_ACK_RANGE        *ack_ranges;
-    size_t                      num_ack_ranges;
+    uint64_t                    num_ack_ranges;
```

Comment:
On the other hand, this should definitely be `size_t`, since it's the length of the `ack_ranges` array.

Reply:
I'll change that back and see what the CI says